packages/editor/src/components/post-text-editor/index.js

```diff
@@ -45,68 +50,45 @@ export class PostTextEditor extends Component {
 	 *
 	 * @param {Event} event Change event.
 	 */
-	edit( event ) {
-		const value = event.target.value;
-		this.props.onChange( value );
-		this.setState( { value, isDirty: true } );
-	}
+	const edit = ( event ) => {
+		const v = event.target.value;
```

Comment:
There is no need to truncate the param name. Use `value` as before for better readability.

Reply:
There already exists a variable _value_ in the outer scope. This causes a linting issue. I have renamed it to _newValue_ for better readability.